cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/DefaultPreviewRunnerManager.java

```diff
@@ -133,7 +136,8 @@ public void terminated(State from) {
           previewRunnerServices.remove(pollerService);
           if (previewRunnerServices.isEmpty()) {
             try {
-              stop();
```

Comment:
Should just call `stop()`. The use of terminator is unnecessary

Reply:
done.